src/drop.js

```diff
@@ -10,6 +10,9 @@ var slice = require('./slice');
  * Acts as a transducer if a transformer is given in list position.
  * @see R.transduce
  *
+ * Dispatches to its second argument's `slice` method if present. As a
+ * result, one may replace `[a]` with `String` in the type signature.
```

Comment:
While this looks like a very promising way to deal with the Strings-often-act-as-lists issue, I'm afraid it might become tedious in our source code.  I'm all for it now.  But let's keep an eye on it.